Fundamentals restructuring (#10929)
aspnet/Docs: aspnetcore/index.md

We're working hard to close the API gap from .NET Framework to .NET Core. The [Windows Compatibility Pack](/dotnet/core/porting/windows-compat-pack) made thousands of Windows-only APIs available in .NET Core. These APIs weren't available in .NET Core 1.x.

## Recommended learning path

We recommend the following sequence of tutorials and articles for an introduction to developing ASP.NET Core apps:

1. Follow a tutorial for the type of app you want to develop or maintain:

   |App type  |Scenario  |Tutorial  |
   |----------|----------|----------|
   |Web app       | For new development        |[Get started with Razor Pages](xref:tutorials/razor-pages/razor-pages-start) |

>    |Web app       | For maintaining an MVC app |[Get started with ASP.NET Core MVC](xref:tutorials/first-mvc-app/start-mvc)|

Reviewer: ```suggestion
   |Web app       | For maintaining an MVC app |[Get started with MVC](xref:tutorials/first-mvc-app/start-mvc)|
```

   |Web API       |                            |[Create a web API](xref:tutorials/first-web-api)\*  |
   |Real-time app |                            |[Get started with SignalR](xref:tutorials/signalr) |

1. Follow a tutorial that shows how to do basic data access:

   |Scenario  |Tutorial  |
   |----------|----------|
   | For new development        |[Razor Pages with Entity Framework Core](xref:data/ef-rp/intro) |

>    | For maintaining an MVC app |[ASP.NET MVC with EF Core](xref:data/ef-mvc/intro)

Reviewer: ```suggestion
   | For maintaining an MVC app |[MVC with Entity Framework Core](xref:data/ef-mvc/intro)
```

1. Read an overview of ASP.NET Core features that apply to all app types:

   * [Fundamentals](xref:fundamentals/index)

1. Browse the Table of Contents for other topics of interest.

> \* There is a new [web API tutorial that you follow entirely in the browser](https://docs.microsoft.com/en-us/learn/modules/build-web-api-net-core), no local IDE installation required.  The code runs in an [Azure Cloud Shell](https://azure.microsoft.com/features/cloud-shell/), and [curl](https://curl.haxx.se/) is used for testing.

Reviewer: ```suggestion
\* There is a new [web API tutorial that you follow entirely in the browser](https://docs.microsoft.com/learn/modules/build-web-api-net-core), no local IDE installation required.  The code runs in an [Azure Cloud Shell](https://azure.microsoft.com/features/cloud-shell/), and [curl](https://curl.haxx.se/) is used for testing.
```
